KlingonAssistant/data/mem-14-r.xml

```diff
@@ -543,11 +543,11 @@ According to a message on the {s.k:src} newsgroup, the first element of this wor
       <column name="entry_name">raw</column>
       <column name="part_of_speech">v:t_c</column>
       <column name="definition">lightning</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">blitzen</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{pe'bIl:n}, {tuD:v}, {jev:v:1}</column>
-      <column name="notes"></column>
+      <column name="notes">Although glossed as a noun, all weather terms are verbs, so this is the verb meaning "it is lightning".</column>
```

Comment:
Not quite happy with the wording. Not all weather "terms" are verbs (e.g., {pe'bIl}). Will think about it and add the same note to related weather verbs.